presto-main/src/main/java/com/facebook/presto/server/BasicQueryInfo.java

```diff
@@ -89,6 +93,7 @@ public BasicQueryInfo(QueryInfo queryInfo)
         this(queryInfo.getQueryId(),
                 queryInfo.getSession(),
                 queryInfo.getState(),
+                queryInfo.getErrorCode() != null ? StandardErrorCode.toErrorType( queryInfo.getErrorCode().getCode()) : null,
```

Comment:
Based on the 2nd review comment, I have added another field for ErrorType in QueryInfo and using this field to populate BasicQueryInfo though its a duplicity of data. So shall I still add this static method?